manifest.yml

```diff
@@ -1,5 +1,5 @@
 name: "FreeRTOS-Plus-TCP"
-version: "V4.0.0"
+version: "V4.1.0"
 description:
```

Comment:
Is the kernel version same? Are we not updating to the latest kernel version?

Reply:
Updated.